docs/java-rest/high-level/getting-started.asciidoc

```diff
@@ -46,8 +46,29 @@ Central]. The minimum Java version required is `1.8`.
 The High Level REST Client is subject to the same release cycle as
 Elasticsearch. Replace the version with the desired client version.
 
-If you are looking for a SNAPSHOT version, the Elastic Maven Snapshot repository is available
-at https://snapshots.elastic.co/maven/.
+If you are looking for a SNAPSHOT version, you should add our snapshot repository to your maven config:
+
+["source","xml",subs="attributes"]
+--------------------------------------------------
+<repositories>
+    <repository>
+        <id>es-snapshots</id>
+        <name>elasticsearch snapshot repo</name>
+        <url>https://snapshots.elastic.co/maven/</url>
+    </repository>
+<repositories>
+--------------------------------------------------
+
+or in Gradle:
+
+["source","groovy",subs="attributes"]
+--------------------------------------------------
+maven {
+        url "https://snapshots.elastic.co/maven/"
+      }
+--------------------------------------------------
+
+Please note that the snapshot repository isn't browsable, but it will work to resolve artifacts correctly.
```

Comment:
I wonder about including this statement. The worry I have is what happens when we make it browsable (something we’ve talked about doing and really should do). Who remembers to come back here and update this note? I think moving the URLs out of the main text of the page so it’s no longer clickable and into configuration helps to reduce the tendency to try to browse the link. Maybe we can see if this addresses the problem without including a high-maintenance note?